scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -253,22 +315,32 @@ workflow case_gatk_acnv_workflow {
   }
 }
 
-# Pad the target file.  This was found to help sensitivity and specificity.  This step should only be altered
-#  by advanced users.  Note that by changing this, you need to have a PoN that also reflects the change.
+# Pad the target file. This was found to help sensitivity and specificity. This step should only be altered
+# by advanced users. Note that by changing this, you need to have a PoN that also reflects the change.
 task PadTargets {
-    File target_bed
+    String target_bed
     Int padding = 250
     File gatk_jar
+    Boolean isWGS
     Int mem
 
+    # Note that when isWGS is true, this task is still called by the workflow.
+    # In that case, to satisfy the output requirements an empty target file is created, that gets 
```

Comment:
In that case, an empty target file is created and passed to the CalculateTargetCoverage task to satisfy input requirements.